doc/level1.rst

```diff
@@ -1026,6 +1027,229 @@ Example Copy when a Reply can't be Encrypted
     future messages in this thread will be encrypted.
 
 
+.. _autocrypt-gossip-example:
+
+Example Autocrypt Gossip headers
+++++++++++++++++++++++++++++++++++
+
+Alice send an e-mail to Bob and Carol with Autocrypt Gossip headers.
+
+::
+
+    Delivered-To: <bob@autocrypt.example>
+    From: Alice <alice@autocrypt.example>
+    To: Bob <bob@autocrypt.example>, Carol <carol@autocrypt.example>
+    Subject: an Autocrypt Gossip header example
+    Autocrypt: addr=alice@autocrypt.example; keydata=
+      mQGNBFn+L+YBDAC3jsOXmFKwKfUh/WxaOErSMMdL1NJzzFCDf4oo0XD5b4ldfVGP09PsNXg5bzUW
```

Comment:
this keydata seems significantly shorter than the one below. is that just 2048 bits?

Reply:
it is copied from the e-mail example above.

inspecting them both suggests that the keydata is truncated somehow :(

Reply:
I'll update both of them, replacing them with the public key derived from the Autocrypt Setup Message.

Reply:
:+1:

Reply:
Also, i think the gossip message needs to be encrypted to Alice, Bob, and Carol, not just to Bob and Carol, since iirc, we recommend "encrypt to self".  That will also make it decryptable just from the setup message example.

@juga0 volunteered to work on this.